### PR TITLE
[19432] Add CONTRIBUTING to repository

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**

### CODE REVIEW

File: CONTRIBUTING [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- [Fast DDS-Gen](https://github.com/eProsima/Fast-DDS-Gen/issues/new)
+- [Fast DDS Python](https://github.com/eProsima/Fast-DDS-python)
+- [Foonathan memory vendor](https://github.com/eProsima/foonathan_memory_vendor/issues/new)
+- [Fast CDR](https://github.com/eProsima/Fast-CDR/issues/new)
+- [Shapes Demo](https://github.com/eProsima/ShapesDemo/issues/new)
+- [ROS 2 Fast DDS RMW](https://github.com/ros2/rmw_fastrtps/issues/new)
+- [Fast DDS Statistics Backend](https://github.com/eProsima/Fast-DDS-statistics-backend/issues/new)
+- [Fast DDS Monitor](https://github.com/eProsima/Fast-DDS-monitor/issues/new)
+- [DDS Router](https://github.com/eProsima/DDS-Router/issues/new)
+
```
Comment (Mario-DL) [inline CONTRIBUTING:49]: Should we include here the [Fast DDS QoS Profiles Manager](https://github.com/eProsima/Fast-DDS-QoS-Profiles-Manager) ?

**Review by Mario-DL — COMMENTED** [review]
Leaving just a question

File: CONTRIBUTING (hunk 1 above)
Comment (JLBuenoLopez-eProsima) [reply CONTRIBUTING:49]: I do not think so. That project is best left out and forgotten :see_no_evil:

**Review by Mario-DL — APPROVED** [review]
LGTM

**Pull request merged**